src/Turbo/README.md

```diff
@@ -21,6 +21,9 @@ Install this bundle using Composer and Symfony Flex:
 ```sh
 composer require symfony/ux-turbo
 
+# install turbo itself
+yarn add @hotwired/turbo --dev
```

Comment:
This looks like something for flex!

Reply:
Oh... dang, you're right! That's an interesting idea!

Reply:
For now, I *do* think we should merge this (as a peer dependency is more correct). But then we can add a new mechanism to Flex that would make this line redundant.

Reply:
can be removed now, see https://github.com/symfony/flex/pull/764

while at it, please check all package.json in this repo :)